src/settings/CopyrightStatusSettings.js

```diff
@@ -29,7 +29,15 @@ class CopyrightStatusSettings extends React.Component {
         baseUrl="coursereserves/copyrightstatuses"
         records="copyrightStatuses"
         label={intl.formatMessage({ id: 'ui-courses.objectName.copyrightstatuses' })}
-        labelSingular={intl.formatMessage({ id: 'ui-courses.objectName.copyrightstatus' })}
+        translations={
+          {
+            cannotDeleteTermHeader: "ui-courses.cv.patronGroup.cannotDeleteTermHeader",
```

Comment:
Here and elsewhere, can you please use single quotes (in accordance with standard Stripes style)? `yarn lint` should have told you about this.

Reply:
fixed! Thanks